examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -196,6 +197,26 @@ public String create(PipelineOptions options) {
       }
     }
 
+    /**
+     * Return default input file path according to runner type.
+     *
+     * <p><ul>
+     *   <li>SparkRunner:
+     *   .../src/test/resources/LICENSE</li>
+     *   <li>other runners:
+     *   gs://dataflow-samples/apache/LICENSE</li>
+     * </ul>
+     */
+    public static class InputFactory implements DefaultValueFactory<String> {
+      @Override
+      public String create(PipelineOptions options) {
+        if (options.getRunner().isAssignableFrom(SparkRunner.class)) {
```

Comment:
Do we want to have everyone use GCS by default?

What if Dataflow was the only one that used the GCS one?

Also, this sets a poor precedent where there is "runner" specific configuration being done on a per test basis.